org.eclipse.xtext.relengutil/META-INF/MANIFEST.MF

```diff
@@ -6,7 +6,18 @@ Bundle-Version: 2.38.0.qualifier
 Bundle-Vendor: Eclipse Xtext
 Bundle-RequiredExecutionEnvironment: JavaSE-17
 Require-Bundle: org.eclipse.xtext.util;bundle-version="2.38.0",
- org.eclipse.m2e.maven.runtime;bundle-version="3.9.600"
+ org.eclipse.m2e.maven.runtime;bundle-version="3.9.600",
```

Comment:
m2e.maven.runtime registers a SLF4J thingy by means of a service loader. Thus we need the command line `-D` when running the mwe2 workflows